site/docs-md/cordova/using-cordova-plugins.md

```diff
@@ -16,16 +16,18 @@ npx cap sync
 Similar to the installation steps. Capacitor will detect the new version of the plugin and install it.
```

Comment:
I don't like the wording of "Capacitor will detect the new version of the plugin" as it's not true. Better say something like, "just update the npm dependency and run update/sync to make Capacitor pick up the changes" or something like that.